datadog_checks_dev/datadog_checks/dev/tooling/commands/release/trello/testable.py

```diff
@@ -144,8 +145,17 @@ def pick_card_member(config: dict, author: str, team: str) -> Optional[str]:
 @click.argument('target_ref')
 @click.option('--milestone', help='The PR milestone to filter by')
 @click.option('--dry-run', '-n', is_flag=True, help='Only show the changes')
+@click.option('--release-version', help='If specified, cards in RC builds column will be updated with this'
```

Comment:
Would it make sense for this to be a command like `testable` instead of a flag? In order to run this command, we still need to include a BASE_REF and TARGET_REF which doesn't seem very related to this flag.

```
$ ddev release trello testable 7.20.0 7.20.1 -n --release-version 7.20.0
Getting diff between '7.20.0' and '7.20.1'... Success!
7.20.0 is an invalid release version. A valid release version is for example 7.21.0-rc.3
```

Reply:
> Would it make sense for this to be a command like testable instead of a flag

When I run the `testable` command to create Trello cards, I also want to update the release version for cards in RC build column most of the time. I can create a new command if you prefer, but creating Trello cards will require to run 2 commands instead of 1.

> In order to run this command, we still need to include a BASE_REF and TARGET_REF which doesn't seem very related to this flag.

Instead of using the flag `release-version`, I can try using `TARGET_REF` and add an optional flag to disable the updates of the card in RC builds (in case TARGET_REF format is not a version as defined by the regex https://github.com/DataDog/integrations-core/pull/7082/files#diff-c1c4e46f59c83c3805dec0bae8068d92R13)

Reply:
```
$ ddev release trello testable 7.20.0 7.20.1 -n --release-version 7.20.0
Getting diff between '7.20.0' and '7.20.1'... Success!
7.20.0 is an invalid release version. A valid release version is for example 7.21.0-rc.3
```
I should also accept `7.21.0` as a valid release version.

Reply:
@ChristineTChen : I have removed `--release-version`. Let me know if it address your comment.